### DISCUSSION TITLE
Client registration without email adresses

### DISCUSSION THREAD

**Discussion by anisryad** [PrestaShop/PrestaShop#35581] [Q&A] [answered]
Hi there, i am very new in this very intersting project,
so i want to know how to permit to a client to checkout without entering email in the form.
the most important part of my client don't have email adress and prefere to use the phone.

how could i bypass the form filling and proceed directly to the delivery.

thanks

**Comment by Hlavtox** [comment, marked as answer]
@anisryad Hi, you pretty much can't. Customer's email is a key part to customer integrity in Prestashop, there just has to be one.

You would need to hard modify the prestashop core on many places.

**Reply by anisryad** [reply]
Hi Daniel,
Ok 
is there a way to replace the email with the phone numbre instead ?
thanks

**Comment by anisryad** [comment]
Good morning,
We would like to be able to replace the email field with a phone number field because most of our future customers may not want to fill in this field if they are in email mode, however all targeted users are used to using their phone number to place orders, this is why we would like to replace this field without modifying the internal functioning of the application by automatically adding (on the fly) @ at the end of the phone number entered such as for example: 0123456789 becomes 0123456789@something.xyz before posting the form.

Or another possibility is to make the email field optional and add another one as phone number field which will be mandatory.

Thank you so much for your help.

**Comment by Maofree** [comment]
Hi
I have a client who asked me the possibility to log in using the phone number or email. Even with the phone they have a way to contact the client for promotions on whatsapp like in this website

![1](https://github.com/user-attachments/assets/1ecae4d3-b490-4059-a208-255780047104)

bye
